client/signup/steps/plans/test/index.jsx

```diff
@@ -124,14 +129,15 @@ describe( 'Plans.onSelectPlan', () => {
 		} );
 	} );
 
-	[
+	test.each( [
 		PLAN_BUSINESS_MONTHLY,
 		PLAN_BUSINESS,
 		PLAN_BUSINESS_2_YEARS,
 		PLAN_ECOMMERCE,
 		PLAN_ECOMMERCE_2_YEARS,
-	].forEach( ( plan ) => {
-		test( `Should add is_store_signup to cartItem.extra when processing wp.com business and eCommerce plans (${ plan })`, () => {
+	] )(
+		`Should add is_store_signup to cartItem.extra when processing wp.com business and eCommerce plans (%s)`,
+		( plan ) => {
 			const myProps = {
```

Comment:
I wonder if we could improve those cases a little bit - e.g. to render the component and run tests on it, rather than in this synthetic way with the raw react component instance.

Reply:
Agreed. There's another PR which is quite similar. I'll follow up 👍🏻